release-tests/pkg_wheel/test_preload.py

```diff
@@ -28,7 +28,11 @@ def test_nccl(self):
             return
         preload_version = self._get_config()['nccl']['version']
         major, minor, patchlevel = (int(x) for x in preload_version.split('.'))
-        expected_version = major * 1000 + minor * 100 + patchlevel
+        major_mult = 1000
+        if major >= 2 and minor > 8:
```

Comment:
Let's get ready for NCCL 3.0 😄 
```suggestion
        if (2, 8) <= (major, minor):
```